### PR TITLE
docs: add note on indexers

### PR DESCRIPTION
**Pull request by cristianmtr**

### CODE REVIEW

File: docs/fundamentals/executor/indexers.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+In the above case, the Storage could be the [PostgreSQL](https://github.com/jina-ai/executors/tree/main/jinahub/indexers/storage/PostgreSQLStorage)-based Storage, while the Query Flow could be based on [FaissPostgresSearcher](https://github.com/jina-ai/executors/tree/main/jinahub/indexers/searcher/compound/FaissPostgresSearcher).
+
+For a showcase code, check our [integration tests](https://github.com/jina-ai/executors/tree/main/tests/integration/psql_dump_reload).
+
+The split between indexing and search Flows allows you to continuously serve requests in your application (in the search Flow), while still being able to write or modify the underlying data. Then when you want to update the state of the searchable data for your users, you perform a dump and rolling update.
+
+### Dump and Rolling Update
+
+The communication between index and search Flows is done via this pair of actions.
+The **dump** action tells the indexers to export its internal data (from whatever format it stores it in) to a disk location, optimized to be read by the shards in your search Flow.
```
Comment (alexcg1) [inline docs/fundamentals/executor/indexers.md:31]: - tells the indexer~s~

File: docs/fundamentals/executor/indexers.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+         'shards': shards,
+         'timeout': -1,
+     },
+ )
+```
+
+where
+
+- `flow_storage` is the Flow with the storage Indexer
+- `target_peapod` is the name of the executor, defined in your `flow.yml`
```
Comment (alexcg1) [inline docs/fundamentals/executor/indexers.md:51]: - **E**xecutor

File: docs/fundamentals/executor/indexers.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+- `dump_path` is the path (on local disk) where you want the data to be stored. **NOTE** The folder needs to be empty. Otherwise, the dump will be cancelled. 
+- `shards` is the nr of shards you have in your search Flow. **NOTE** This doesn't change the value in the Flow. You need to keep track of how you configured your search Flow
```
Comment (alexcg1) [inline docs/fundamentals/executor/indexers.md:53]: - n**umbe**r

File: docs/fundamentals/executor/indexers.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+For performing the **rolling update**, we can see the usage in the same test:
+
+```python
+flow_query.rolling_update(pod_name='indexer_query', dump_path=dump_path)
+```
+
+where
+
+- `flow_query` is the Flow with the searcher Indexer
+- `pod_name` is the name of the executor, defined in your `flow.yml`
```
Comment (alexcg1) [inline docs/fundamentals/executor/indexers.md:64]: **E**xecutor

**Review by alexcg1 — CHANGES_REQUESTED** [review]
Very nice and super readable! Just a few tiny nitpicks

File: docs/fundamentals/executor/indexers.md (hunk 1 above)
Comment (cristianmtr) [reply docs/fundamentals/executor/indexers.md:31]:
```suggestion
The **dump** action tells the indexer to export its internal data (from whatever format it stores it in) to a disk location, optimized to be read by the shards in your search Flow.
```

File: docs/fundamentals/executor/indexers.md (hunk 2 above)
Comment (cristianmtr) [reply docs/fundamentals/executor/indexers.md:51]:
```suggestion
- `target_peapod` is the name of the Executor, defined in your `flow.yml`
```

File: docs/fundamentals/executor/indexers.md (hunk 3 above)
Comment (cristianmtr) [reply docs/fundamentals/executor/indexers.md:53]:
```suggestion
- `shards` is the number of shards you have in your search Flow. **NOTE** This doesn't change the value in the Flow. You need to keep track of how you configured your search Flow
```

File: docs/fundamentals/executor/indexers.md (hunk 4 above)
Comment (cristianmtr) [reply docs/fundamentals/executor/indexers.md:64]:
```suggestion
- `pod_name` is the name of the Executor, defined in your `flow.yml`
```

**Review by alexcg1 — APPROVED** [review]
LGTM :+1:

**Pull request merged**